index.html

```diff
@@ -526,14 +526,15 @@ <h4>Script Type</h4>
           </pre>
         </section>
         <section>
-          <h4>Primary Language</h4>
+          <h4>Languages</h4>
```

Comment:
This will change section linking, and link to here like line 2340 (copied below) need to be updated.
`This is the profile expression of <a href="#primary-language"></a>.`